performance-tests/remotewb-performance-tests/src/main/resources/logback.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
In general, we try to only use `logback-test.xml` files, and also exclude `logback.xml` files from the jars. 

Logback will use the first `logback.xml` or `logback-test.xml` file that it encounters in the classpath: this means that if this jar is in the classpath of another project, it will be impossible to override the settings in this file.